guice/src/main/java/com/flipkart/gjex/http/interceptor/HttpFilterInterceptor.java

```diff
@@ -91,19 +92,23 @@ public final void doFilter(ServletRequest request, ServletResponse response,
                     .stream().collect(Collectors.toMap(String::toLowerCase, httpServletResponse::getHeader));
                 filters.forEach(filter -> filter.doProcessResponseHeaders(responseHeaders));
                 responseHeaders.forEach(responseWrapper::setHeader);
+                filters.forEach(filter -> filter.doProcessResponse(responseWrapper));
```

Comment:
This will cause filter thrown exception to be caught. We don't want that. Filter thrown error should not be caught